packages/api-bindings/src/apollo/createRevenueAggregateTypePolicy.ts

```diff
@@ -0,0 +1,15 @@
+import { never } from '@lens-protocol/shared-kernel';
+
+import { erc20Amount, RevenueAggregate } from '../graphql';
+import { TypePolicy } from './TypePolicy';
+
+export function createRevenueAggregateTypePolicy(): TypePolicy<RevenueAggregate> {
+  return {
+    fields: {
+      totalAmount(_, { readField }) {
+        const total = readField('total') ?? never('total is null');
```

Comment:
oh, that's interesting that we don't have to use aliased name `__total` and it can properly map `total` to `__total`

Reply:
Yeah, this is not very clear in the Apollo docs... but my understanding is that FieldPolicy applies to field as in "edges of the GQL original nodes"... so if a field is aliased more than once in a given query the field policy will be called as any times as it's number of aliases. The 2nd argument of the field read method should have details of the field actual query including the alias name I think, but I didn't investigate further.